website/content/commands/connect/envoy.mdx

```diff
@@ -227,6 +231,34 @@ This example assumes that the correct [environment variables](#api-options) are
 used to set the local agent connection information and ACL token, or that the
 agent is using all-default configuration.
 
+### Sidecar proxy with ACLs enabled
+
+In secure deployments, Consul's ACL system is enabled with a default `deny` policy.
+To access Consul API resources, an API request must present a Consul ACL token
+with the necessary privileges. A sidecar proxy must be provided with an ACL token
+with privileges to register itself and the service it fronts,
+and to access all potential upstreams of that service.
```

Comment:
```suggestion
with the necessary privileges. You must add an ACL token to the sidecar proxy’s configuration when ACLs are enabled. The token must grant the proxy privileges to register itself and the service it fronts and to access all potential upstreams of that service.
```
I think this was my most recent suggestion. After rereading it I can see that it is ambiguous. I think this is clearer.

Reply:
I went with something substantially similar:

> To access Consul API resources, an API request must present a Consul ACL token with the necessary privileges. If ACLs are enabled, you must provide an ACL token to the sidecar proxy. The token must grant the proxy privileges to register itself and the service it fronts and to access all potential upstreams of that service.